packages/browser/package.json

```diff
@@ -29,23 +29,18 @@
     "prepack": "pnpm test"
   },
   "dependencies": {
-    "@logto/js": "^1.0.0-alpha.2",
-    "@silverhand/essentials": "^1.1.6",
-    "jose": "^4.5.0",
-    "lodash.get": "^4.4.2",
-    "lodash.once": "^4.1.1",
-    "superstruct": "^0.16.0"
+    "@logto/client": "^1.0.0-alpha.2",
+    "@silverhand/essentials": "^1.1.6"
   },
   "devDependencies": {
     "@jest/types": "^27.5.1",
+    "@logto/js": "^1.0.0-alpha.2",
```

Comment:
why do we need to add `@logto/js` as dev dependency?

Reply:
Used for mockups, make it easy to test `storage`

Reply:
created LOG-3636 to refactor in the future.